functional_tests/scylla_operator/test_functional.py

```diff
@@ -87,6 +93,128 @@ def test_single_operator_image_tag_is_everywhere(db_cluster):
+        pod_data = {namespace: {}}
+        for pod_name_and_ip in pod_names_and_ips:
+            pod_name, pod_ip = pod_name_and_ip.split()
+            pod_data[namespace][pod_name.strip()] = pod_ip.strip()
+            assert pod_ip not in ("", "None"), "Pod IPs were expected to be set"
+        return pod_data
+
+    log.info('Deploy first ScyllaCluster')
+    try:
+        log.debug(db_cluster.k8s_cluster.helm_install(
+            target_chart_name=target_chart_name, source_chart_name="scylla-operator/scylla",
+            version=operator_version, use_devel=True, values=values, namespace=namespace))
+        db_cluster.k8s_cluster.kubectl_wait("--all --for=condition=Ready pod", namespace=namespace, timeout=1200)
+
+        # NOTE: check that Scylla services are headless - no IPs are set
+        svc_ips = kubectl(
+            "get svc --no-headers -o=custom-columns=':.spec.clusterIP'"
+            f" -l scylla/cluster={cluster_name} -l scylla-operator.scylladb.com/scylla-service-type=member",
+            namespace=namespace).stdout.split()
+        assert svc_ips
+        assert len(svc_ips) == 3
+        assert all(svc_ip in ('', 'None') for svc_ip in svc_ips), "SVC IPs were expected to be absent"
+
+        # NOTE: read Scylla pods IPs
+        pods_data = get_pod_names_and_ips(cluster_name=cluster_name, namespace=namespace)
+        try:
+            create_namespace(db_cluster=db_cluster, namespace=namespace2)
+            db_cluster.k8s_cluster.create_scylla_manager_agent_config(namespace=namespace2)
+
+            log.info('Deploy second ScyllaCluster')
+            values.set("datacenter", f"{dc_name}-2")
+            values.set("externalSeeds", list(pods_data[namespace].values()))
+            values.set("fullnameOverride", cluster_name2)
+            log.debug(db_cluster.k8s_cluster.helm_install(
+                target_chart_name=target_chart_name2, source_chart_name="scylla-operator/scylla",
+                version=operator_version, use_devel=True, values=values, namespace=namespace2))
+            db_cluster.k8s_cluster.kubectl_wait(
+                "--all --for=condition=Ready pod", namespace=namespace2, timeout=1200)
+            pods_data |= get_pod_names_and_ips(cluster_name=cluster_name2, namespace=namespace2)
+            all_pod_ips = {}
+            for current_namespace in pods_data.keys():
+                for pod_ip in pods_data[current_namespace].values():
+                    all_pod_ips[pod_ip] = dc_name + "" if current_namespace == namespace else "-2"
+
+            log.info("Verify DB cluster's peers info")
+            for current_namespace in pods_data.keys():
+                for pod_name, pod_ip in pods_data[current_namespace].items():
+                    cqlsh_cmd = "select peer, data_center, rack, rpc_address from system.peers;"
+                    cqlsh_output = parse_cqlsh_select_output(kubectl(
+                        f"exec {pod_name} -- /bin/cqlsh -e \"{cqlsh_cmd}\"",
```

Comment:
we are doing this since we don't have a node object for that those ones ? i.e. using cqlsh like this ?

Reply:
Yes, for now the approach we have for creating DB cluster and DB node objects is not flexible.
So, I used this way of getting data.

Reply:
o.k.